content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md

```diff
@@ -87,7 +92,17 @@ To add an entity under a prefab instance as an override:
 
 In the below image, an Antenna entity has been added to the first Car prefab instance.
 
-![Adding an entity as an override.](/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-add.png)
+{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-add.png" width="750" alt="Adding an entity as an override." >}}
+
+## Delete an entity or a nested prefab as an override
+
+To delete an entity or a nested prefab instance as an override:
+1. In Entity Outliner, **right-click** on an entity or a nested prefab instance and choose **Delete** from the context menu.
+1. Notice that a blue circle appears on the parent entity's icon in Entity Outliner. This indicates that a child has been deleted as an override.
+
+{{ <note> }}
+Once deleted as an override, there is only one way of reverting this operation, which is through an edit undo. Currently, you cannot revert on the parent entity. See the GitHub issue [#13437](https://github.com/o3de/o3de/issues/13437) for more details.
```

Comment:
Is it worth mentioning that there's no UX to revert the deletion at this time, but another (more advanced) way is to hand edit the .prefab file to remove the override? I'm not sure if that would be too advanced for a user guide/tutorial.

Reply:
Good suggestion. I'll add that as well.